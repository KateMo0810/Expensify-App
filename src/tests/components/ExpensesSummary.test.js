import React from 'react';
import {shallow} from 'enzyme';
import {ExpensesSummary} from '../../components/ExpensesSummary';

test('Should correctly render ExpensesSummary with 1 expense',() =>{
    const wrapper = shallow(<ExpensesSummary  expenseCount={1} expensesTotal={145} />);
    expect(wrapper).toMatchSnapshot();
});

test('Should correctly render ExpensesSummary with multiple expenses',() =>{
    const wrapper = shallow(<ExpensesSummary  expenseCount={3} expensesTotal={12345} />);
    expect(wrapper).toMatchSnapshot();
});