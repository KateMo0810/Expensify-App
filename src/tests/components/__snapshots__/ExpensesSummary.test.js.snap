exports[`test Should correctly render ExpensesSummary with 1 expense 1`] = `
<div>
  <h4>
    Viewing 
    1
     
    expense
     totalling 
    $1.45
  </h4>
</div>
`;

exports[`test Should correctly render ExpensesSummary with multiple expenses 1`] = `
<div>
  <h4>
    Viewing 
    3
     
    expenses
     totalling 
    $123.45
  </h4>
</div>
`;
